build-release-tools/application/nightly_build_clean_up.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+"""
+Nightly build will only be stored for two weeks.
+This is a command line program that clean up the overdue nightly builds.
```

Comment:
please add Note in comments:
```
Note:
the release build will never be deleted by this script.
because 
get_build_date_from_version() will return NULL for bintray/dockerhub sub routine, and "continue" to skip deletion.
for vagrant,  release build starts from 1. and old builds are only two digit(0.1x), so the sub-routine will skip the deletion also.


```